# ava_slam_data.msg

geometry_msgs/Point eulerAngle0
geometry_msgs/Point eulerAngle1
geometry_msgs/Point eulerAngle2
geometry_msgs/Point eulerAngle3
geometry_msgs/Point position0

nav_msgs/Odometry pose0
nav_msgs/Odometry pose1
nav_msgs/Odometry pose2
nav_msgs/Odometry pose3
sensor_msgs/Imu imu0
sensor_msgs/Imu imu1
sensor_msgs/Imu imu2
sensor_msgs/Imu imu3

geometry_msgs/Point simulationTime
geometry_msgs/PointStamped initialTime
